packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -197,12 +198,8 @@ const helpers = {
               ...contentNegotiationResult,
             }));
         } else {
-          logger.trace(`Unable to find a content for ${mediaTypes}, returning an empty text/plain response.`);
-          return right({
-            code,
-            mediaType: 'text/plain',
-            headers: headers || [],
-          });
+          logger.warn(`Unable to find a content for ${mediaTypes}`);
+          return left(ProblemJsonError.fromTemplate(NOT_ACCEPTABLE, `Unable to find a content for ${mediaTypes}`));
```

Comment:
is it ok to say `a content`? ain't that uncountable?

Reply:
Ah, I do not know :) — what'd the be correct sentence?

Reply:
just content is ok, no need for a.